### PR TITLE
live: add link to youtube stream

### PR DESCRIPTION
**Pull request by rissson**

### CODE REVIEW

**Review by addict3d — APPROVED** [review]

File: live.html [hunk 1]
```diff
@@ -10,6 +10,7 @@ <h3>NixCon Live</h3>
         <div class="container">
             <p>To watch the stream from VLC on your computer, run:<p>
             <code>nix-shell --packages vlc --run "vlc https://dash.nixcon.net/dash/master.m3u8"</code>
+            <p>To watch past talks, go <a href="https://www.youtube.com/watch?v=7sQa04olUA0" target="_blank">here</a>.<p>
```
Comment (ghuntley) [inline live.html]: Suggest changing "To watch past talks" to something like "Also available on YouTube (which supports rewind) if you want to see previous talks"

Comment (rissson) [reply live.html]: Done. Merged that suggestion with the one you made on IRC.

**Review by ghuntley — APPROVED** [review]
Amazing. nb: I haven't tested it.

**Comment by rissson** [comment]
> Amazing. nb: I haven't tested it.

image

**Pull request merged**